repo-agent/issue-sandbox/main.go

```diff
@@ -119,31 +121,31 @@ func processGitChanges(oldCommitID string) error {
 	// Commit and push
 	if githubUserEmail != "" {
 		// Check if there are any changes to commit
-		statusOutput, err := runCommand("git", "status", "--porcelain")
+		statusOutput, err := runGitCommand("status", "--porcelain")
```

Comment:
Idea: you might consider a package `git` with a type `Repo`, because then you don't need to worry about which directory you are in etc